collective/contentrules/mailtogroup/actions/mail.py

```diff
@@ -96,16 +103,75 @@ def __init__(self, context, element, event):
         self.context = context
```

Comment:
@fulv  here at line [100](https://github.com/collective/collective.contentrules.mailtogroup/pull/9/files#diff-4c9070de35a2ba0217cedb5665f50917R100) There are two inheritance of Interface class, we can remove one, right ?